azure-pipelines.yml

```diff
@@ -110,8 +106,8 @@ jobs:
     - script: sh -x ./clean_build.sh
       displayName: Clean Build
     - task: PublishBuildArtifacts@1
-      displayName: Publish Release DMG
-      condition: eq(variables['B_BUILD_TYPE'], 'Release')
+      displayName: Publish Release DMG if on Big Sur agent
```

Comment:
It will be apparent in azure whether the step is run or not, so we don't need  to embed the condition in the description.

Reply:
Fixed in ea38ae65.

Reply:
I don't have that line in my local `azure-pipelines.yml`.

Reply:
Looking at the PR commit tab, this line isn't present at all, so I'm going to close this comment. It must have been fixed elsewhere. It's also not in my commit history.